iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/StaticVarCompensatorAdderImpl.java

```
@@ -31,6 +31,8 @@ class StaticVarCompensatorAdderImpl extends AbstractInjectionAdder<StaticVarComp
 
     private TerminalExt regulatingTerminal;
 
+    private boolean regulating;
```

If you initialize regulating to false, then all code that builds a static var compensator needs to be modified.
For example in the code below you must add setRegulating(true) or it will no longer work as expected.
The compiler is of no help here and silent errors may remain.
Is it really the desired behavour ? It is quite dangerous.
If yes this is insufficiently documetned in the current version of the  migration guide.
      svc1.setBmin(-0.002)
                .setVoltageSetpoint(385)
                .setRegulationMode(StaticVarCompensator.RegulationMode.VOLTAGE)
                **.setRegulating(true);**